extras/upgrade_activities.sh

```diff
@@ -81,7 +84,19 @@ function check_daemons_running()
     my_exit 1
   fi
 
-  if ! ps cax | grep -wq '[g]luster-blockd'
+  # try 5 times to make sure the backgroud process is fired.
+  count=1
+  while (($count <= 5))
```

Comment:
In all the places where we use a variable please use ${my_var} syntax, flower braces to maintain consistency across the file.

Reply:
Also, it will be nice to use the same syntax in other places where we check for something like
while [[ ${count} -le 5 ]]
...

if [[ ${count} -gt 5]]
...

Reply:
Sure, will fix it.